library/src/main/java/com/datatorrent/lib/io/ConsoleOutputOperator.java

```diff
@@ -53,13 +56,14 @@
     public void process(Object t)
     {
       String s;
+      PrintStream stream = isStderr() ? System.err : System.out;
```

Comment:
Why not declare `stream` as `transient` variable and initialize it in setup and when ever stder is changed assign `stream` accordingly. Doing this for every tuple is not good...